bem-css/style.css

```diff
@@ -76,7 +76,7 @@ body {
   color: #FFF;
 }
 
-.label__field {
+.input {
   display: block;
```

Comment:
Great thanks for this PR, @tadatuta!
Can you just fix one small issue with regular `input` styles (see below) and I will merge!
Also, if we already started working on layout 'holder' fixes, maybe you look at BEM-tree for nav and propose how we can separate layout column and navigation?
Thanks!